src/xdocs/google_style.xml

```diff
@@ -1168,7 +1168,7 @@
                             <td><a name="5.2.7"/><a href="#5.2.7"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s5.2.7-local-variable-names">5.2.7 Local variable names</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s5.2.7-local-variable-names">5.2.7 Local variable names</a>
```

Comment:
Simplified rule for local variables. They should be lowerCamelCase and obey to no further restrictions. This is exactly what the current check does.